src/components/organisms/Header/index.tsx

```diff
@@ -0,0 +1,36 @@
+import { AppBar, Box } from '@mui/material';
+import { Logo } from '../../atoms/Logo';
+import Navbar from '../../molecules/Navbar';
+import { PlusButton } from '../../atoms/PlusButton';
+import { HeartButton } from '../../atoms/HeartButton';
+
+export const Header = () => (
+  <AppBar
+    position="static"
+    sx={{
+      alignItems: 'center',
+      padding: '20px 0 16px',
```

Comment:
you can use rem here. or https://mui.com/material-ui/customization/spacing/ 
<img width="886" alt="image" src="https://github.com/KaterynaLikhvar/dog-app-init/assets/15706494/7c0d665d-9771-41e5-b504-cf8555e362dd">
https://mui.com/system/getting-started/the-sx-prop/#spacing